src/main/scala-2.12/kamon/scala/instrumentation/FutureInstrumentation.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * =========================================================================================
+ * Copyright © 2013-2014 the kamon project <http://kamon.io/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ * =========================================================================================
+ */
+
+package kamon.scala.instrumentation
+
+import kamon.trace.{TraceContextAware, Tracer}
+import org.aspectj.lang.ProceedingJoinPoint
+import org.aspectj.lang.annotation._
+
+@Aspect
+class FutureInstrumentation {
```

Comment:
I think that this file is not necessary because the above contains contains the same mixins and poincuts and also the divition between 2.11 and 2.12. Sorry if i'm missing something :(. @ivantopo WDYT?